This adds last commit info to `quarto check` for 99.9.9 versions.
Adds info when quarto doesn't have the right version of Pandoc/Dart Sass.
Also upgrades info to warning.
--- a/src/command/check/check.ts
+++ b/src/command/check/check.ts
@@ -5,7 +5,7 @@
 *
 */
 
-import { info } from "log/mod.ts";
+import { info, warning } from "log/mod.ts";
 
 import { render, renderServices } from "../render/render-shared.ts";
 
@@ -70,11 +70,13 @@ async function checkVersions(_services: RenderServices) {
       throw new Error(`Unable to determine ${name} version`);
     }
     if (!satisfies(version, constraint)) {
-      info(
+      warning(
         `      NOTE: ${name} version ${version} is too old. Please upgrade to ${
           constraint.slice(2)
         } or later.`,
       );
+      const name_env = "QUARTO_"+name.toUpperCase().replace(" ","_");
+      warning(`            Or set ${name_env} to a path of an appropriate ${name} version`);
     } else {
       info(`      ${name} version ${version}: OK`);
     }
@@ -113,6 +115,9 @@ async function checkVersions(_services: RenderServices) {
 async function checkInstall(services: RenderServices) {
   completeMessage("Checking Quarto installation......OK");
   info(`      Version: ${quartoConfig.version()}`);
+  if (quartoConfig.version() === "99.9.9"){
+    info(`      Commit:  ${quartoConfig.commit()}`);
+  }
   info(`      Path: ${quartoConfig.binPath()}`);
   if (Deno.build.os === "windows") {
     try {
--- a/src/core/quarto.ts
+++ b/src/core/quarto.ts
@@ -40,6 +40,14 @@ export const quartoConfig = {
       return kLocalDevelopment;
     }
   },
+  commit: () => {
+    const commitPath = join(getenv("QUARTO_SHARE_PATH"), "commit");
+    if (existsSync(commitPath)) {
+      return Deno.readTextFileSync(commitPath);
+    } else {
+      return "";
+    }
+  },
   dotenv: async (): Promise<DotenvConfig> => {
     if (!dotenvConfig) {
       const options: LoadOptions = {
